feature-group-definitions/webrtc-sctp.yml

```
@@ -0,0 +1,11 @@
+name: WebRTC SCTP information
+description: The `sctp` object on `RTCPeerConnection` provides information about the negotiated SCTP transport. SCTP is the protocol used by `RTCDataChannel`.
```

@ddbeck do you think https://github.com/web-platform-dx/web-features/tree/main/docs#provides applies here?

---

I don't think it does, though if you want to dodge it:

```suggestion
description: The `sctp` object on `RTCPeerConnection` represents the negotiated SCTP transport. SCTP (Stream Control Transmission Protocol) is the protocol that `RTCDataChannel` uses.
```

I suggest expanding the abbreviation here, _unless_ it's not widely known by its unabbreviated form (e.g., I probably wouldn't spell out API or TCP/IP).

---

I think that "represents" is accurate, although the its read-only nature is more clear from "information about". But I like it better overall.